src/SFA.DAS.Forecasting.Application/Infrastructure/Registries/ConfigurationHelper.cs

```diff
@@ -50,6 +50,11 @@ public static CommitmentsApiConfig GetCommitmentsApiConfiguration()
                 {
                     BaseUrl = CloudConfigurationManager.GetSetting("Commitments-BaseUrl"),
                     ClientToken = CloudConfigurationManager.GetSetting("Commitments-ClientToken"),
+                    ApiBaseUrl = CloudConfigurationManager.GetSetting("Commitments-ApiBaseUrl"),
```

Comment:
These will have to be changes as well to follow the  new app settings

Reply:
Ok